SSA/Projects/InstCombine/HackersDelight.lean

```diff
@@ -143,13 +143,13 @@ theorem and_ule_not_xor :
     x &&& y ≤ᵤ ~~~(x ^^^ y) := by
   sorry
 
-def AdditionNoOverflows? (x y : BitVec w) : Prop := (x.adc y false).1
+def additionNoOverflows? (x y : BitVec w) : Prop := (x.adc y false).1
```

Comment:
This should have been capitalised. Oh well. Too late now.

Reply:
https://leanprover-community.github.io/contribute/naming.html)](https://leanprover-community.github.io/contribute/naming.html

Rule 2

Reply:
You can always remove it from the queue and fix things. 😄